indexer/src/main/java/org/pdxfinder/utils/CbpTransformer.java

```diff
@@ -0,0 +1,152 @@
+package org.pdxfinder.utils;
+
+
+import org.pdxfinder.dataexport.UniversalDataExporter;
+import org.pdxfinder.graph.dao.Group;
+import org.pdxfinder.services.OmicTransformationService;
+import org.pdxfinder.services.UtilityService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@Service
+public class CbpTransformer {
+
+    private static final Logger log = LoggerFactory.getLogger(CbpTransformer.class);
+
+    private UtilityService utilityService;
+    private OmicTransformationService omicTransformationService;
+    private UniversalDataExporter universalDataExporter;
+
+    CbpTransformer(UtilityService utilityService, OmicTransformationService omicTransformationService, UniversalDataExporter universalDataExporter){
+        this.utilityService = utilityService;
+        this.omicTransformationService = omicTransformationService;
+        this.universalDataExporter = universalDataExporter;
+    }
+
+    private static String notSpecified = "Not Specified";
+    private static String patientId = "patientId";
+    private static String sampleId = "sampleId";
+    private static String entrezGeneId = "EntrezGeneId";
+
+    public enum cbioType {
+        MUT,
+        GISTIC
+    }
+
+    public void exportCBP(File exportDir,File templateDir, File pathToJson, cbioType dataType) throws IOException {
+
+        if (doesFileNotExist(exportDir) || doesFileNotExist(templateDir) || doesFileNotExist(pathToJson)) {
+            throw new IOException("A string argument passed to the exportCBP does not point to an existing file.");
+        }
+            Group jsonGroup = createGroupWithJsonsFilename(pathToJson.getAbsolutePath());
+
+            List<Map<String, Object>> listMapTable = utilityService.serializeJSONToMaps(pathToJson.getAbsolutePath());
+            cbpMapsToSheetsByDataType(listMapTable, dataType);
+
+            universalDataExporter.setDs(jsonGroup);
+            universalDataExporter.setTemplateDir(templateDir.getAbsolutePath());
+            universalDataExporter.export(exportDir.getAbsolutePath());
+    }
+
+    public void convertListOfEntrez(List<String> entrezIds){
+        entrezIds.forEach(e ->
+            log.info(omicTransformationService.ncbiGeneIdtoHgncSymbol(e))
```

Comment:
This method does not appear to do anything other than log - you want to clarify the intention in the method name or perform the functionality outside of the `log.info()` call